superglue/package.json

```diff
@@ -9,7 +9,10 @@
     "copy:package": "cat ./package.json | jq 'del(.scripts)' > dist/package.json",
     "copy:readme": "cp ../README.md dist/",
     "build:index": "babel lib -d dist",
```

Comment:
```suggestion
```

Reply:
Lets remove this line since `build` no longer uses it.

Reply:
Done!